cache/file/file_test.go

```diff
@@ -0,0 +1,84 @@
+package file
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/pierrre/imageserver"
+	imageserver_cache "github.com/pierrre/imageserver/cache"
+	cachetest "github.com/pierrre/imageserver/cache/_test"
+	"github.com/pierrre/imageserver/testdata"
+	"io/ioutil"
+	"os"
+)
+
+var _ imageserver_cache.Cache = &Cache{}
+var tmpDirPath string
```

Comment:
I would name this variable `testDirPath`.
I add the `test` prefix to all global variable/function in the tests.